Makefile

```diff
@@ -22,8 +23,19 @@ vm-sanity-checks:
 platform-sanity-checks:
 	cd platform_sanity_checks && ${MAKE} ${PASS_DOWN_ARGS}
 
+rmsr/rmsr.ko:
+ifeq ($(shell uname -s),Linux)
+	# Allow git to fail if rmsr has already been cloned.
+	${GIT} clone https://github.com/softdevteam/rmsr.git | true
```

Comment:
There isn't a `build.sh` in Krun. We can push this out to a separate shell script (in which case it still needs to be run from the Makefile) or make the `rmsr` repository a submodule of Krun.

I can't think of any other options...